opal/mca/rcache/base/rcache_base_vma_tree.c

```diff
@@ -131,6 +132,17 @@ mca_rcache_base_vma_item_t *mca_rcache_base_vma_new (mca_rcache_base_vma_module_
     return vma_item;
 }
 
+/* NTH: this function MUST not allocate or deallocate memory */
```

Comment:
How do we know thatOBJ_RELEASE is not putting the refcount to 0 (and free the memory) ?